tests/store/test_emissions.py

```diff
@@ -108,10 +123,33 @@ def test_read_file():
     assert metadata.items() >= expected_metadata.items()
 
 
-# TODO: Add test for adding additional years data - 2013 gpp cardomom
+    # TODO: Add test for adding additional years data - 2013 gpp cardomom
 
+@pytest.fixture
+def load_edgar(bucket):
+    def _load_edgar(version, year, database_version: bool = True):
```

Comment:
There's a few assumptions here we can't necessarily make around EDGAR data
 - For the database version there was previously a v4.3.2 and there may be other versions going forward which use similiar syntax
 - We may want to look at monthly edgar data rather than yearly (that's a case we will need to add in future)
 - We may also want to look at a different species e.g. co2 or no2

Overall, I think we'd be better off just allowing this be flexibly called rather than setting this up as a more clever fixture.

Reply:
And think this isn't an update which is needed in this PR

Reply:
This fixture is really only meant for the tests in that file. I added it because otherwise I would have had to make the standardize changes in 10 places instead of 1. I can substitute the body of the fixture back into the tests, but I still need to update those tests one way or another.

Reply:
Sure, I'm thinking about building in flexibility which will be needed in this test file (other versions / gases for edgar will likely be included in tests within this file).

A few small additions which may address this:
 - make the full string for the version be included in the input (solves the "v4.3.2" issue)
 - add species as an input (e.g. "ch4")